### DISCUSSION TITLE
css global file

### DISCUSSION THREAD

**Discussion by cybermedusa** [sveltejs/kit#5977] [General]
Hi,
I've made a separate file for styling named global.css and I've put it inside the routes folder then I imported it inside __layout.svelte file (import './app.css';) and my styling is not working. Is there a different way to have a global file for styling instead of typing everything inside the component <style> tag?
All of my elements in global.css have this ":global" typing.

**Comment by ruudtimmermans** [comment]
What i usually do in the style tag in my layout file
```html
<style>
    :global {
        @import '../lib/styles/app.css';
    }
</style>`

**Reply by unlocomqx** [reply]
wait, that should be style not script

**Comment by JLAcostaEC** [comment]
You don't need to use :global on separated css file, move your styles to SRC folder and import from there in your layout script, other option is use svelte-preprocess to import that file correctly

**Reply by cybermedusa** [reply]
thanks, it worked
